pytest_playwright/pytest_playwright.py

```diff
@@ -371,3 +376,9 @@ def pytest_addoption(parser: Any) -> None:
         choices=["on", "off", "only-on-failure"],
         help="Whether to automatically capture a screenshot after each test.",
     )
+    group.addoption(
+        "--full-page-screenshot",
```

Comment:
would it be better to make this a `store_true` boolean flag? with only the 2 options available, would simplify the logic when checking it too i.e `if config.option.full_page_screenshot:`

Reply:
I think that would be a better option. I was following the convention of `--screenshot` since this is basically a sub option. Updating now!